examples/README.md

```diff
@@ -0,0 +1,179 @@
+
+Copy the generated token somewhere safe, it will be used in the next steps.
+
+## Configure Astarte to be accessible (for the local instance)
+
+We will assume that the machine running the Astarte local instance and the ESP32 are connected
+to the same local network.
+
+First, identify your local IP address (on Linux run `ip addr`).
+Then, in the folder of the Astarte installation, locate the file: `compose.env`.
+Open it and modify the following line:
+
+`PAIRING_BROKER_URL=mqtts://localhost:8883/` to `PAIRING_BROKER_URL=mqtts://<MY_IP_ADDR>:8883/`
+
+Furthermore, change also the following line:
+
+`VERNEMQ_ENABLE_SSL_LISTENER=true` to `VERNEMQ_ENABLE_SSL_LISTENER=false`
+
+After these changes, you will have to restart your astarte instance for them to take effect.
+
+## Configuring the astartectl tool (for the remote instance)
+
+Most of the provided examples will make use of the `astartectl` tool to interact with the remote
+instance of astarte.
+This tool needs an access token to interact with the remote Astarte instance. We can find this
+token in the *Astarte Cloud Console* (the page on Astarte Cloud where all your realms are shown) on
+the *Realm Information* page.
+The token is presented as a command that will store it in an environment variable. Something similar
+ to:
+```
+TOKEN="token_content"
+```
+Run this command once in a shell to set the `TOKEN` environment variable for a single shell
+ session.
+
+## Configuring the SDK
+
+We will assume you are running the ESP-IDF from the command line. If you are using the GUIs provided
+with the Visual Studio Code or the Eclipse plugins then the configuration procedure might be
+slightly different.
+
+Open the configuration tool using the following command:
+| ESP-IDF $\geqslant$ v4.x | ESP-IDF v3.x |
+| ------------- | ------------ |
+| `idf.py menuconfig` | `make menuconfig` |
+
+In the *Astarte datastreams example* tab fill in:
+- *WiFi SSID* the SSID of your wifi network
+- *WiFi Password* the SSID of your wifi network
+- *Credentials secret* (only for manual device registration) the device-specific credential secret
+generated in the previous step
+- *Device hardware ID* (only for manual device registration) the device hardware ID generated in the
+previous step
+
+In the *Component config ---> Astarte SDK* tab fill in:
+- *Astarte realm* place here the name of your Astarte realm
+- *Astarte pairing base URL* use the Base API URL for your remote/local Astarte instance. Adding at
+the end `/pairing` in case of the remote instance. (e.g.: if you are using Astarte Cloud:
+`https://api.eu1.astarte.cloud/pairing`, or if you are using the local Astarte instance
+`http://<MY_IP_ADDR>:4003`).
+
+**NB**: This will place the above-mentioned properties in the ESP-IDF sdk project configuration.
+Cleaning the configuration will also reset them to their default values. If you want to store them
+more permanently you can overwrite the defaults by adding them in the `sdkconfig.defaults` file.
```

Comment:
I'd say "by adding them _to_ the `sdkconfig.defaults`" (also [Chat-GPT dixit](https://telegra.ph/Chat-GPT-dixit-02-22))